Gemfile

```diff
@@ -43,7 +43,7 @@ gem 'activerecord-import'
 gem 'progressbar'
 
 # RailsAdmin is a Rails engine that provides an easy-to-use interface for managing your data
-gem 'rails_admin', '~> 1.4'
+gem 'rails_admin', '~> 2.2.1'
```

Comment:
We could probably be a little more lenient and do 2.2 here, so we could updated to any 2.x version in the future.